### PR TITLE
Phishing campaign & ML - added documentation regarding XSIAM

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
Fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-34760

## Description
- Added documentation that explains the phishing campaign and ML content are only supported on Cortex XSOAR.
- Excluded the `Phishing - Machine Learning Analysis` playbook from XSIAM, and configured the task to skip it if unavailable (in Phishing - Generic v3)

### CODE REVIEW

File: Packs/Phishing/ReleaseNotes/3_6_13.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+#### Playbooks
+
+##### Phishing - Generic v3
+
+- Updated the documentation to state what platforms support the machine-learning content used in the playbook.
+
+##### Phishing - Machine Learning Analysis
+
+- Updated the documentation to state what platforms support the machine-learning content used in the playbook.
```
Comment (ShirleyDenkberg) [inline Packs/Phishing/ReleaseNotes/3_6_13.md:10]:
```suggestion
Updated the documentation to state what platforms support the machine-learning content used in the playbook.

##### Phishing - Machine Learning Analysis

Updated the documentation to state what platforms support the machine-learning content used in the playbook.
```

**Comment by ShirleyDenkberg** [comment]
@AdiPeret Doc review completed.

File: Packs/Phishing/Playbooks/Phishing_-_Machine_Learning_Analysis.yml
Comment (ssokolovich) [inline Packs/Phishing/Playbooks/Phishing_-_Machine_Learning_Analysis.yml:1]: maybe its worth adding the `marketplace: xsoar` key in this PB?

Comment (idovandijk) [reply Packs/Phishing/Playbooks/Phishing_-_Machine_Learning_Analysis.yml:1]: yes

File: Packs/Phishing/ReleaseNotes/3_6_13.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+##### Phishing - Generic v3
+
+- Updated the documentation to state what platforms support the machine-learning content used in the playbook.
+- The playbook will now skip the *Phishing - Machine Learning Analysis* playbook when it is unavailable.
+
+##### Phishing - Machine Learning Analysis
+
+- Updated the documentation to state what platforms support the machine-learning content used in the playbook.
+- The playbook is now unavailable in XSIAM due to the automations used in it.
```
Comment (ssokolovich) [inline Packs/Phishing/ReleaseNotes/3_6_13.md:12]:
```suggestion
- The playbook is now available only for the XSOAR marketplace.
```

**Review by ssokolovich — APPROVED** [review]

File: Packs/Phishing/ReleaseNotes/3_6_13.md (hunk 2 above)
Comment (idovandijk) [reply Packs/Phishing/ReleaseNotes/3_6_13.md:12]: Better

**Comment by idovandijk** [comment]
@ShirleyDenkberg can you please review again? Made a change

**Pull request merged**